toolflow/scala/src/main/scala/tapasco/base/builder/Builds.scala

```diff
@@ -33,6 +33,37 @@ import scala.io.Source
   * json lib documentation for details).
   */
 private[tapasco] trait Builds[A] {
+
+  private def buildPathString(nodes: List[PathNode]): String = nodes.map(
+    x => "\"" + x.toString.replace("/", "") + "\""
+  ).mkString("/")
+
+  def errorHandling(json:JsValue, error: JsError, basePath: Option[Path] = None): Unit = {
+    val _logger = tapasco.Logging.logger(getClass)
+    val origin = (json \ "DescPath").get.toString()
+    for(entry <- error.errors) {
+      val path = buildPathString(entry._1.path)
+      val prefix = "Json Error in File %s at Json-Path %s:".format(origin, path)
+      entry._2.foreach(x => x.messages.foreach {
+        case "error.path.missing" => _logger.warn("%s Entry is missing.".format(prefix))
```

Comment:
Are this strings (e.g. `error.path.missing`) generated by the JSON library or by our code? In the latter case, I would prefer enums over string matching.

Reply:
They are not generated by the library, but the library uses the same approach to encode all other errors. That is why i have implemented it like this.

Reply:
I would prefer to either use enums or to have the strings defined in a single location (`val` inside a `Object`), so that changes to the string have to be made in a single place only. 

With strings defined in multiple places that need to match, maintainability suffers.